### PR TITLE
add 17 icons

### PR DESCRIPTION
**Pull request by SiMah5**
## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### updated
* RustDesk
* Obsidian
* Tiktok 
* Joplin
* V2RayNG

### Icons added from icon request form
* Pix You Cornflower Dark Icons (`com.arjunarora.pix_you_cornflower_dark_icons/com.arjunarora.pix_you_cornflower_dark_icons.MainActivity`)
* NYTimes (`com.nytimes.android/com.nytimes.android.MainActivity`)
* MPlayer (`com.mplayer.streamcast/com.mplayer.streamcast.activity.SplashScreen`)
* MoveInSync (`com.moveinsync.ets/com.moveinsync.ets.launch.LaunchActivity`)
* Gosund (`com.gosund.smart/com.gosund.smart.splash.SplashActivity`)
* Goodreads (`com.goodreads/com.goodreads.kindle.ui.activity.RoutingActivity`)
* Finch (`com.finch.finch/com.finch.finch.MainActivity`)
* Obedience (`com.obedience/com.obedience.DefaultAlias`)
* Drinkable (`com.moimob.drinkable/com.moimob.drinkable.MainActivity`)
* Focus Go (`com.franco.focus.lite/a.b`)
* Amaze Utilities (`com.amaze.fileutilities/com.amaze.fileutilities.home_page.MainActivity`)
* Andronix (`studio.com.techriz.andronix/studio.com.techriz.andronix.ui.Splash`)
* Weather (`org.breezyweather/org.breezyweather.main.MainActivity`)
* TG Monet (`com.c3r5b8.telegram_monet/com.c3r5b8.telegram_monet.MainActivity`)

### Icons added 
* Element X dbg (`io.element.android.x.debug/io.element.android.x.MainActivity`)
* Kafka (`com.kafka.user/com.kafka.user.MainActivity`)
* BeauTyXT (`dev.soupslurpr.beautyxt/dev.soupslurpr.beautyxt.MainActivity`)

### CODE REVIEW

File: svgs/gosund.svg
Comment (x9136) [inline svgs/gosund.svg:1]: Why did you decide not to make a circle like in the original icon? 
gosund

Comment (SiMah5) [reply svgs/gosund.svg:1]: oh... i think i forgot to change the corner radius when i made it bigger so it turned out like that, my bad. thanks for pointing it out.

File: svgs/kafka.svg
Comment (Chefski) [inline svgs/kafka.svg:1]: The icon goes outside the 160x160px content area. Could you reduce it in size to fit the content area?

File: svgs/tg_monet.svg
Comment (Chefski) [inline svgs/tg_monet.svg:1]: The icon goes slightly outside the content area
image

**Review by Chefski — CHANGES_REQUESTED** [review]

**Review by Chefski — APPROVED** [review]

**Pull request merged**